src/legacy/core_plugins/console/public/quarantined/_app.scss

```diff
@@ -1,6 +1,16 @@
 // TODO: Move all of the styles here (should be modularised by, e.g., CSS-in-JS or CSS modules).
 #consoleRoot {
-  height: 100%;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: stretch;
+  -ms-flex-align: stretch;
+  align-items: stretch;
```

Comment:
is this needed? i _think_ the default behavior is `stretch`

https://developer.mozilla.org/en-US/docs/Web/CSS/align-items